mllib/src/test/scala/org/apache/spark/ml/regression/LinearRegressionSuite.scala

```diff
@@ -34,6 +34,7 @@ class LinearRegressionSuite extends SparkFunSuite with MLlibTestSparkContext {
   private val seed: Int = 42
   @transient var dataset: DataFrame = _
   @transient var datasetWithoutIntercept: DataFrame = _
+  @transient var datasetWithManyFeature: DataFrame = _
 
```

Comment:
Also, changed `dataset` into `datasetWithDenseFeature`, and `datasetWithoutIntercept` into `datasetWithDenseFeatureWithoutIntercept`.